packages/libs/error-stack/src/fmt/hook.rs

```diff
@@ -492,12 +494,43 @@ mod default {
         });
     }
 
+    #[cfg(feature = "pretty-print")]
+    enum LocationDisplay<'a> {
+        Color(&'a Location<'static>),
+        None(&'a Location<'static>),
+    }
```

Comment:
I'd prefer this to be a struct containing `location: &'a Location<'static>` and an enum with the color variants instead. That avoids having the same inner variant on all enum variants.

Reply:
fixed in [e3cb384](https://github.com/hashintel/hash/pull/1794/commits/e3cb38419de0d10e7e93f7c80c0721701fe95606)